src/Symfony/Bridge/Twig/Form/TwigRendererEngine.php

```diff
@@ -29,12 +29,20 @@ class TwigRendererEngine extends AbstractRendererEngine implements TwigRendererE
      */
     private $template;
 
+    public function __construct(array $defaultThemes = array(), \Twig_Environment $environment)
```

Comment:
people using it have to instantiate it explicitly. This signature change would break Silex for instance